src/jdk.jartool/share/classes/sun/security/tools/jarsigner/Resources.java

```diff
@@ -157,6 +157,7 @@ public class Resources extends java.util.ListResourceBundle {
                 "The jar will be treated as unsigned, because it is signed with a weak algorithm that is now disabled.\n\nRe-run jarsigner with the -verbose option for more details."},
         {"jar.treated.unsigned.see.weak.verbose",
                 "WARNING: The jar will be treated as unsigned, because it is signed with a weak algorithm that is now disabled by the security property:"},
+        {"multiple.manifest.warning.", "Duplicate manifest entries were detected in the jar file. JarSigner will operate on the first one found and others will be discarded."},
```

Comment:
I actually don't know which one will be operated on. If you are sure it is the first one, can you add some lines in the test to check this?

Reply:
There is another test which checks this behavior: `test/jdk/tools/jar/MultipleManifestTest.java`. Also, it's not really in the scope of the change we're testing here. I've modified the message to be less specific.